machAeroTutorials/airfoilopt_mesh.rst

```diff
@@ -53,9 +53,9 @@ General Options
 ``unattachedEdgesAreSymmetry``
     Tells pyHyp to automatically apply symmetry boundary conditions to any unattached edges (those that do not interface with another block).
 ``outerFaceBC``
-    Tells pyHyp to which boundary condition to apply to the outermost face of the extruded mesh (Note that we do not set the inlet or outlet boundaries seperately because it is automatically handled in adflow by the free stream flow direction).
+    Tells pyHyp which boundary condition to apply to the outermost face of the extruded mesh. Note that we do not set the inlet or outlet boundaries seperately because they are automatically handled in ADflow consistently with the free stream flow direction.
 ``BC``
-    Tells pyHyp that since it is a 2D problem, the jlow and jhigh  are set to be symmetry boundary conditions on both sides of the domain (Note that Klow=1 is set by default to be a non-slip wall).
+    Tells pyHyp that, since it is a 2D problem, both sides of the domain ``jLow`` and ``jHigh``  are set to be symmetry boundary conditions. Note that ``kLow=1`` is set by default to be a non-slip wall.
```

Comment:
I think it should be just ``kLow`` not ``kLow=1``?

Reply:
thanks for catching it!

Reply:
I was looking into the ``BC`` option as part of [this issue](https://github.com/mdolab/pyhyp/issues/27). I found that there is actually no ``kLow`` or ``kHigh`` option. Also, I wouldn't say it's no-slip by default because you can use the same mesh for Euler runs. I think it's more accurate to say that the input surface mesh (which corresponds to ``kLow``) is automatically assigned to be a wall.

Reply:
Very good point... We should stay as general as possible! Would you mind edit this really quick? Otherwise I might do this tomorrow

Reply:
I just realized that I can't push to this branch.

As a broader issue, we should discuss what to document in these tutorials vs. what to document in the code repos. There are some options documented here that are not documented in `pyHyp`. There is also duplication in this repo for the airfoil and wing cases. After this PR, the airfoil mesh generation page will have details that the wing mesh generation page does not.

Reply:
Do you want me to edit and push it ?

Reply:
That would be great.

Reply:
Okay I did that!

Reply:
Just ``kLow`` was correct. Marco already made that edit. I would replace the entire second sentence with "The input surface is automatically assigned to be a wall boundary."

Reply:
I agree on the "misallocation" of the info throughout the codes and the repos @sseraj . I was looking for some of these options on `pyHyp` too and got a bit confused. We probably need a major check of the tutorial and the related repos,.. yet another problem connected to the bigger "knowledge transfer" issue. We need to discuss this soon with the maintainers for every repo!

Reply:
There is bound to be some duplicated information, but the most comprehensive and general ones (such as documenting the options) should be in the repository docs, and I think in most cases it's sufficient to point readers to those pages without duplicating the explanation. The information here should be specific to this particular case.

Reply:
I agree. I will add the ones here to the pyHyp options table.